client/src/styles.scss

```diff
@@ -8,28 +8,20 @@
 // Be sure that you only ever include this mixin once!
 @include mat.core();
 
-// TODO(v15): As of v15 mat.legacy-core no longer includes default typography styles.
-//  The following line adds:
-//    1. Default typography styles for all components
-//    2. Styles for typography hierarchy classes (e.g. .mat-headline-1)
-//  If you specify typography styles for the components you use elsewhere, you should delete this line.
-//  If you don't need the default component typographies but still want the hierarchy styles,
-//  you can delete this line and instead use:
-//    `@include mat.legacy-typography-hierarchy(mat.define-typography-config());`
-@include mat.all-component-typographies();
```

Comment:
Can you say something about the impacts of removing this line? I can see that you did some things with typography in later parts of the file. Thanks!

Reply:
This section was added in a1801e1c7b5ea5756baa4300a72ad5ee5a27a059 as part of the Angular Material migrations but had 
```scss
@include mat.all-legacy-component-typographies();
```
This was needed to provide typography theming because the theme defintions would still only contain color information and not typography. 

While in 19c287e718b3a20360a360fc4349e6843afdea86 it was replaced with 
```scss
@include mat.all-component-typographies();
```
as part of the migration to MDC components, the theme definitions were not updated.

I have now updated the theme definitions to include typography so this is not needed anymore as its included in `mat.all-component-themes()` when typography is included in the theme object.